tools/python/maps_generator/generator/env.py

```diff
@@ -328,49 +460,29 @@ def _osm_tools_setup():
         logger.info("Build osm tools ...")
         return build_osmtools(settings.OSM_TOOLS_SRC_PATH)
 
-    @staticmethod
-    def _out_path_setup(build_name):
-        dt = datetime.datetime.now()
-        version_format = "%Y_%m_%d__%H_%M_%S"
-        if build_name:
-            dt = datetime.datetime.strptime(build_name, version_format)
-
-        s = dt.strftime(version_format)
-        mwm_version = dt.strftime("%y%m%d")
-        planet_version = int(dt.strftime("%s"))
-
-        out_path = os.path.join(settings.MAIN_OUT_PATH, s)
-        Env._create_if_not_exist(settings.MAIN_OUT_PATH)
-        Env._create_if_not_exist(out_path)
-        return out_path, s, mwm_version, planet_version
-
-    @staticmethod
-    def _intermediate_path_setup(out_path):
-        intermediate_path = os.path.join(out_path, "intermediate_data")
-        Env._create_if_not_exist(intermediate_path)
-        return intermediate_path
-
-    @staticmethod
-    def _tmp_dir_setup():
-        Env._create_if_not_exist(settings.TMPDIR)
-        os.environ["TMPDIR"] = settings.TMPDIR
-
-    @staticmethod
-    def _create_if_not_exist(path):
-        try:
-            os.mkdir(path)
-            logger.info(f"Create {path} ...")
-            return True
-        except FileExistsError:
-            return False
-
-    def _prepare_borders(self):
-        borders = "borders"
-        temp_borders = os.path.join(self.intermediate_path, borders)
-        Env._create_if_not_exist(temp_borders)
-        borders = os.path.join(settings.USER_RESOURCE_PATH, borders)
+    def setup_borders(self):
+        temp_borders = self.paths.generation_borders_path
+        # It needs in a case of rebuilding several mwms.
```

Comment:
s/needs/is needed/
s/in a case/in case/

Reply:
++